docs/devices/EER50000.md

```diff
@@ -12,19 +12,32 @@ description: "Integrate your Schneider Electric EER50000 via Zigbee2MQTT with wh
 | Model | EER50000  |
 | Vendor  | Schneider Electric  |
 | Description | Wiser H-Relay (HACT) |
-| Exposes | climate (occupied_heating_setpoint, local_temperature), linkquality |
+| Exposes | ## climate (occupied_heating_setpoint, local_temperature), linkquality |## WRONG !
```

Comment:
This is generated from https://github.com/Koenkk/zigbee-herdsman-converters/blob/3f6e6a7a5b2565ed104be0a4a9c52a8056ef1419/devices/schneider_electric.js#L493, so please update that instead